updates/99-fas.update

```diff
@@ -91,3 +91,9 @@ default:objectClass: groupOfPrincipals
 default:objectClass: ipaKrb5DelegationACL
 default:cn: fasjson-http-delegation
 default:ipaAllowedTarget: cn=ipa-ldap-delegation-targets,cn=s4u2proxy,cn=etc,$SUFFIX
+
+# create the container for the user agreements
+dn: cn=agreements,$SUFFIX
+default: objectclass: top
+default: objectclass: nsContainer
+default: cn: agreements
```

Comment:
The plugin adds some managed permissions. Therefore the update file must also call the update plugin at the end of the file:

```
plugin: update_managed_permissions
```